forward/ft_prepare_vol_sens.m

```diff
@@ -302,7 +302,10 @@
       % don't do anything, h2em or h2mm generated later in ft_prepare_leadfield
 
     case 'duneuro'
-
+      
+      % show the license (only once)
+      duneuro_license;              
```

Comment:
this already happens in ft_hastoolbox, right?

Reply:
yes, removed here 05bfc64 (where I deleted also in other locations, i.e., ft_headmodel_duneuro.m and ft_comute_leadfield.m )